### DISCUSSION TITLE
Discussions are open

### DISCUSSION THREAD

**Discussion by Acly** [Acly/krita-ai-diffusion#384] [Announcements]
## Welcome!
  This is a place for questions, ideas and discussions around using Stable Diffusion in Krita.

  Feel free to share:
  * 🛠️ **Workflows**, how do you incorporate image generation into your work? This can help others and inspire changes in the plugin!
  * 💡 **Ideas** you want to discuss which aren't conrete feature proposals yet
  * ❓ **Questions** on how to achieve certain things with the plugin
  * 📰 **News** about technology that might be relevant to the project
  * 🖼️ **Images** and anything cool you created
  * 📝 **Tutorials** which can then become part of the documentation

**Comment by Danamir** [comment]
This is great, thanks.

**Comment by Muskelmagier** [comment]
this could be interesting 

https://www.reddit.com/gallery/1akyabn

https://github.com/ZHO-ZHO-ZHO/ComfyUI-BRIA_AI-RMBG

**Reply by freecoderwaifu** [reply]
Was meaning to post about RMBG so I'm glad you did. Built in support for background removal with RMBG would be great, while Krita has background removal with Gmic's Extract Foreground, it's not really as good.

**Reply by Acly** [reply]
There is #12, still think [krita-ai-tools](https://github.com/Acly/krita-ai-tools) would be a better place to implement it, still has to actually be done though...

**Comment by ArshCypherZ** [comment]
i use enhanceai's [image generations](https://enhanceai.art), it got a lot of useful features

**Comment by pjvaz30** [comment]
I would like to see "Layers Prompting" .  For instance, setting one layer with the style, artist, camera ect.  Then as objects are added to the image in subsequent layers they can be locked via the inherit Alpha feature, keeping the image cohesive. (similar to Photoshops Clipping Masks). Typing Prompts directly into the Layers Name might be ideal so as not to lose them when trying something out.  Just a thought.

**Reply by ETCore7** [reply]
A workaround is [GLIGEN GUI](https://github.com/mut-ex/gligen-gui). It has its own webUI. You can copy-paste results into Krita, but it is not producing actual separate layers.

**Comment by ETCore7** [comment]
Maybe a dumb question, but the Canny Edge BW controlnet image in Live mode is ignored as img2img input (as expected), while a Line Art BW controlnet image is not ignored, it makes the result flat and bright. Is this normal?
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/ca3cdfd7-1e0a-41ec-b8e8-fcc96b8a88d5)
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/27b68ffe-114d-4939-89d8-6b80b6dca439)

**Reply by Acly** [reply]
The idea is that line art is often an intentional part of the final image. You can draw it in a transparent layer on top of your colors, in which case it makes sense to use it as both the control layer, and the img2img input.

If you don't want that you need to set the line art layer to invisible (or move it down the stack).

**Comment by ETCore7** [comment]
Thank you for clarifying.

**Comment by KevinDAYAN** [comment]
It seems that python.exe is not multithreaded with this application. How can I activate it to make it faster?

**Comment by lerizo91** [comment]
Hello everyone, I would like to know if you could help me. I have been learning to use Kripta with the AI plugin, and I would like to know if you could advise me on which models to use. I want to add elements to different photos or replace some, trying to achieve a result with good resolution and detail. I have been installing some models, but my internet connection is very poor, and it becomes quite tedious to do simple tests. If you could recommend a configuration and models to use, it would be very helpful. Greetings to everyone, and thank you in advance.

**Comment by HAMSAILY** [comment]
https://github.com/Acly/krita-ai-diffusion/discussions/2018

I don't know what's different between qwen vs wan 2.2  , but  it seems there is something we must be looking for

**Comment by HAMSAILY** [comment]
Is there any way to put fast prompt styles, something to work with, so that you can save your prompt styles or use other prompt styles and edit them?

<img width="1341" height="575" alt="image" src="https://github.com/user-attachments/assets/70666520-d870-43c4-8cc9-7181dd356b44" />
<img width="1200" height="630" alt="image" src="https://github.com/user-attachments/assets/b629cab6-1312-473b-8427-b7083f54d144" />

<img width="2534" height="1017" alt="image" src="https://github.com/user-attachments/assets/321ff5ab-a6a0-4129-89e8-0a7d82b6f2fa" />
examples for styles prompts 

 https://drive.google.com/file/d/1gtIaIGH2mUFcN2QjIOw7wKfRhQw3PDCv/view


https://www.youtube.com/watch?v=UyBnkojQdtU&t=42s

**Comment by HAMSAILY** [comment]
Wan 2.2 Vs Qwen Image   
https://www.youtube.com/watch?v=btln5sJT5VA
